### PR TITLE
Adding ADS QPI extension to gallery

### PR DESCRIPTION
**Pull request by jovanpop-msft**
I'm author of new extension for Azure Data Studio,

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -3074,7 +3074,47 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+                {
+                  "extensionId": "62",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json]: Please also update the TotalCount value at the bottom of this file

File: extensionsGallery.json [hunk 2]
```diff
@@ -3074,7 +3074,47 @@
...
+                      "assetUri": "",
+                      "fallbackAssetUri": "fallbackAssetUri",
+                      "files": [
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+                          "source": "https://github.com/JocaPC/AzureDataStudio-QPI/releases/download/stable/qpi-0.1.6.vsix"
+                        },
+                      {
+                        "assetType": "Microsoft.VisualStudio.Services.Content.Details",
+                      "source": "https://raw.githubusercontent.com/jocapc/AzureDataStudio-QPI/master/README.md"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json]: Please add more information to https://github.com/JocaPC/AzureDataStudio-QPI/blob/master/README.md. See https://raw.githubusercontent.com/microsoft/azuredatastudio/master/extensions/admin-tool-ext-win/README.md for an example. At the very least it should have a brief description of what the extension is and how to use it.

File: extensionsGallery.json [hunk 3]
```diff
@@ -3074,7 +3074,47 @@
...
+                  "versions": [
+                    {
+                      "version": "0.1.6",
+                      "lastUpdated": "03/08/2020",
+                      "assetUri": "",
+                      "fallbackAssetUri": "fallbackAssetUri",
+                      "files": [
+                        {
+                          "assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+                          "source": "https://github.com/JocaPC/AzureDataStudio-QPI/releases/download/stable/qpi-0.1.6.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json]: Direct VSIX installs should be uploaded to our blob storage account. I've done that - the URL to use is `https://sqlopsextensions.blob.core.windows.net/extensions/qpi/qpi-0.1.6.vsix`

File: extensionsGallery.json (hunk 2 above)
Comment (jovanpop-msft) [reply extensionsGallery.json]: Changed readme

File: extensionsGallery.json (hunk 1 above)
Comment (jovanpop-msft) [reply extensionsGallery.json]: Updated

File: extensionsGallery.json (hunk 3 above)
Comment (jovanpop-msft) [reply extensionsGallery.json]: Updated

**Comment by jovanpop-msft** [comment]
@alanrenmsft @Charles-Gagnon I made all changed. I also aligned id=63 both in standard and insider extension.

**Pull request merged**